docs/platform/nodebalancer/getting-started-with-nodebalancers-new-manager/index.md

```diff
@@ -0,0 +1,119 @@
+
+4.  Now we must add the single backend node to the NodeBalancer's configuration. Point this at the private IP address of your web server Linode.
+
+    [![Adding a backend node to a NodeBalancer.](nodebalancers-backend-nodes-small.png "Adding a backend node to a NodeBalancer")](nodebalancers-backend-nodes.png)
+
+    These configuration changes will take a few moments to be reflected by your NodeBalancer. If everything is configured on your backend correctly, once the changes have gone through, the **Node Status** column will update to **1 up / 0 down**.
+
+    [![The backend node has been added, and is now status UP.](nodebalancers-1up-small.png "The backend node has been added, and is now status UP")](nodebalancers-1up.png)
+
+    If the backend status reports **0 up / 1 down**, check to make sure that your web application is configured to respond on the Linode's private IP address.  You do this by adding the private IP address to your /etc/hosts file on your Linode and then reboot your Linode. There might be a virtual host mismatch as well -- check the notes in the next section.
+
+5.  Now that the backend is up, go directly to your NodeBalancer's IP address in a browser. You should see your web application as the NodeBalancer proxies the traffic through.
+
+    [![Viewing the NodeBalancer-driven web site in a browser.](nodebalancers-hello-world.png)](nodebalancers-hello-world.png)
+
+### A Note about Virtual Hosts
+
+You might not see the web application that you expect when you go directly to the NodeBalancer's IP address. This is due to virtual hosts, and is not an issue unique to NodeBalancers. In the default configurations of many web servers, an application might only be configured to respond for certain hostnames. This can impact testing NodeBalancers as well as the behavior of their health checks.
+
+It is important to configure the "default" virtual host in your web server to point at something useful. NodeBalancer will pass the Host header from a browser untouched, so virtual hosts will work entirely normally once you are pointing a domain at the NodeBalancer. It is only mentioned here because testing NodeBalancers can demonstrate quirks in a web server's configuration, particularly when browsing by the NodeBalancer's IP address.
+
+ {{< note >}}
+Health checks are transmitted with a Host header (in HTTP/1.0 mode).
+{{< /note >}}
+
+## Putting the NodeBalancer in Charge
+
+Your NodeBalancer is now working and is able to pass traffic to your web application. It is important to note at this point that configuring the NodeBalancer has not impacted your application's normal operations at all -- you can test NodeBalancer without your users ever knowing. The only exception to this would be when adding a private IP address, if necessary.
```

Comment:
"The only exception to this would be when adding a private IP address, if necessary." <-- this is a little unclear. You can add a private IP to a Linode without affecting anything. Please explain.